lib/ansible/modules/crypto/openssl_csr.py

```diff
@@ -407,10 +445,20 @@ def _check_extenededKeyUsage(extensions):
         def _check_basicConstraints(extensions):
             return _check_keyUsage_(extensions, b'basicConstraints', self.basicConstraints, self.basicConstraints_critical)
 
+        def _check_ocspMustStaple(extensions):
+            oms_ext = [ext for ext in extensions if ext.get_short_name() == b'UNDEF' and ext.get_data() == b'\x30\x03\x02\x01\x05']
```

Comment:
Also re: https://github.com/ansible/ansible/pull/35082/files#r166218363, in what cases are these other magic number needed?

Reply:
A pre-1.1 version of OpenSSL doesn't know about `tlsfeature` and returns this instead. At least, after trying around with the tests, this was the only way to check for this feature on some pyOpenSSL / OpenSSL combinations.